modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta content="IE=edge" http-equiv="X-UA-Compatible" />
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+		<title>Document</title>
```

Comment:
Please change the title to the specific name for this template.

Reply:
Done.